services/postgres/Dockerfile

```diff
@@ -1,3 +1,26 @@
-FROM postgres:13-alpine
+FROM postgres:15-alpine
+
+# Install dcron and su-exec for user switching
+RUN apk update && apk add --no-cache dcron su-exec
+
+# Copy SQL dump and PostgreSQL configuration
 COPY ./scripts/restore-2024-04-25-22-06-17.sql /docker-entrypoint-initdb.d/
-COPY ./postgresql.conf /postgresql.conf
\ No newline at end of file
+COPY ./scripts/postgresql.conf /postgresql.conf
+
+# Create log directory and set permissions
+RUN mkdir -p /var/log/pg_log && \
```

Comment:
Why not let the volume mount do the work for you? You could run cron on the host, too.